editor/editor_resource_picker.cpp

```diff
@@ -922,6 +947,89 @@ void EditorResourcePicker::_ensure_resource_menu() {
 	edit_menu->connect("popup_hide", callable_mp((BaseButton *)edit_button, &BaseButton::set_pressed).bind(false));
 }
 
+void EditorResourcePicker::_gather_resources_to_duplicate(const Ref<Resource> p_resource, TreeItem *p_item, const String &p_property_name) const {
+	p_item->set_cell_mode(0, TreeItem::CELL_MODE_CHECK);
+
+	String res_name = p_resource->get_name();
+	if (res_name.is_empty() && !p_resource->is_built_in()) {
```

Comment:
Note that the check I've suggested originally follows a similar check used in a few places of this file, including a similar resolution for the `assign_button`. But I suppose this should also work.